mlir/utils/jenkins/reportUtils.py

```diff
@@ -4,12 +4,12 @@
 
 from typing import Tuple, List
 
-PERF_REPORT_FILE = 'mlir_vs_miopen_perf.csv'
```

Comment:
All should stay as `MIOpen`.

And the two files in `perfdb` come from MIOpen tuned database. @jerryyin please confirm.